dataframe.cabal

```diff
@@ -14,7 +14,7 @@ maintainer:         mschavinda@gmail.com
 
 copyright: (c) 2024-2025 Michael Chavinda
 category: Data
-tested-with: GHC ==9.8.3 || ==9.6.6 || == 9.4.8 || ==9.10.1 || ==9.12.1 || ==9.12.2
+tested-with: GHC ==9.4.8 || ==9.6.7 || ==9.8.4 || ==9.10.2 || ==9.12.2
```

Comment:
Just sorted these, bumped to latest patch version and removed 9.12.1 (I think it's sufficient to have just 1 job per ghc minor version).